nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/pom.xml

```diff
@@ -53,71 +59,83 @@
-            </plugin>
-            <plugin>
-                <artifactId>maven-resources-plugin</artifactId>
-                <executions>
                     <execution>
                         <id>copy-resources</id>
                         <phase>validate</phase>
                         <goals>
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
-                            <outputDirectory>${project.build.directory}/${project.artifactId}-${project.version}/docs/rest-api/images</outputDirectory>
-                            <resources>          
+                            <outputDirectory>${restApiDirectory}/images</outputDirectory>
+                            <resources>
                                 <resource>
                                     <directory>src/main/resources/images</directory>
                                 </resource>
-                            </resources>              
-                        </configuration>            
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.swagger.core.v3</groupId>
+                <artifactId>swagger-maven-plugin-jakarta</artifactId>
+                <version>2.2.19</version>
+                <configuration>
+                    <resourcePackages>
+                        <package>org.apache.nifi.web.api</package>
+                    </resourcePackages>
+                    <outputPath>${swaggerDirectory}</outputPath>
+                    <outputFileName>swagger</outputFileName>
+                    <outputFormat>JSONANDYAML</outputFormat>
+                    <prettyPrint>true</prettyPrint>
+                    <defaultResponseCode>200</defaultResponseCode>
+                    <configurationFilePath>${project.build.outputDirectory}/openapi.yaml</configurationFilePath>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>resolve</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.swagger.codegen.v3</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>3.0.50</version>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${swaggerDefinition}</inputSpec>
+                            <output>${restApiDirectory}</output>
```

Comment:
Not exactly sure what is happening, but it seems like this is not producing the output in `docs/rest-api` and then it is not ending up in the final assembly. Going to embedded docs and clicking Rest API ends up on a Yikes not found page.

Reply:
Thanks for catching that. I went through a few iterations and apparently the last version did not correctly map the URL to the HTML location, will fix this and push the changes.